selectrum.el

```diff
@@ -268,6 +281,15 @@ If PREDICATE is non-nil, then it filters the collection as in
    (t
     (error "Unsupported collection type %S" (type-of collection)))))
 
+(defun selectrum--count-info ()
+  "Return a string of count information to be prepended to prompt."
+  (let ((total (length selectrum--refined-candidates))
+        (current (1+ (or selectrum--current-candidate-index -1))))
+    (cond ((eq selectrum-count-style 'matches)
```

Comment:
I think `pcase` would be more elegant to use here.

Reply:
Done.